source/includes/steps-3.6-upgrade-sharded-cluster.yaml

```diff
@@ -136,37 +138,43 @@ title: Upgrade the shards.
 level: 4
 ref: 3.6-upgrade-shards
 content: |-
-   Upgrade the shards one at a time. If the shards are replica sets, for each shard:
+   Upgrade the shards one at a time. If you are running standalone
+   shards, first convert them to a
+   :doc:`replica set</tutorial/convert-standalone-to-replica-set>`.
```

Comment:
The location of this conversion seems not quite right.    Since now, my config servers are already 3.6

Reply:
Also, we mention standalone in

https://docs-mongodbcom-staging.corp.mongodb.com/jdestefano/DOCS-11064/reference/method/sh.addShard.html


And, did you test whether if you had a standalone shard, that just changing to the repl set correctly updates the shard meta data held by the config servers?

Reply:
- Sorry, I'm not sure I understand the first comment? Let's discuss tomorrow. 
- Updated sh.addShard and the addShard dbcommand.
- I did not test the meta data scenario. I am not sure how to do this off the top of my head but can look into this tomorrow if needed.